iex_trading/anaconda-project.yml

```diff
@@ -14,6 +14,11 @@ labels:
 skip:
   - IEX_to_CSV.ipynb
 
+user_fields:
```

Comment:
I think this should be made more compact because it will be in every file, right?

```
user_fields: [labels, skip, maintainers]
```

And it also needs copying to the other .yml files, right?